### ISSUE TITLE
examples/sprites: Performance regression on Mac OS X

### ISSUE THREAD

**Issue by hajimehoshi** [hajimehoshi/ebiten#202]
Interestingly, I couldn't find performance regression on browsers.

**Comment by hajimehoshi** [comment]
Made sure that this doesn't happen on 1.2. Let's bisect.

**Comment by hajimehoshi** [comment]
Note: 6e98e0716d75b707f1079b00aa5a1bb3150c959e is good

**Comment by hajimehoshi** [comment]
```
There are only 'skip'ped commits left to test.
The first bad commit could be any of:
06ec244ff3e3fe3ac6ad6bc4136257d4c2f4e683
73856a2298897f1cdabcf205bd21341dc247e71e
We cannot bisect more!
```

**Comment by hajimehoshi** [comment]
The situation seems flaky, but 922f3925446aad51426e7284d25d84b844c6595b seems like a culprit.

**Comment by hajimehoshi** [comment]
Fixed by 3ec0020f026423c36c618994e4cef18c447eec85
